packages/xo-web/src/common/intl/messages.js

```diff
@@ -966,6 +966,8 @@ const messages = {
   // ----- Host actions ------
   disableMaintenanceMode: 'Disable maintenance mode',
   enableMaintenanceMode: 'Enable maintenance mode',
+  slaveHostMoreUpToDateThanMasterAfterReboot:
+    'It appears that your slave host will be more up-to-date than the master after the reboot. This will result in the slave being unable to contact the pool master. Please update and reboot your master node first.',
```

Comment:
```suggestion
    'It appears that this host will be more up-to-date than the master after the reboot. This will result in the slave being unable to contact the pool master. Please update and reboot your master node first.',
```